db/seeds.rb

```diff
@@ -17,3 +17,5 @@
 # t.boolean :parking, default: false
 # t.boolean :food_vendor, default: false
 # t.boolean :bathroom, default: false
+
+Beach.create(street1: "123 Beach Drive", street2: "Apt. 5", city: "Boston", state: "MA", zip: "01234", name: "Tom Ford Beach", user_id: 1)
```

Comment:
Use `find_or_create_by!` as it will look in the database for an entry or create it if it isn't there. Moreover, the `!` lets you know if there any problems when attempting to save an object to the db.